test/init/init.js

```diff
@@ -4,6 +4,7 @@
 // restricted by GSA ADP Schedule Contract with IBM Corp.
 'use strict';
 
+process.env.UV_THREADPOOL_SIZE = 64;
```

Comment:
unfortunately, I couldn't use the `mocha-test` simple script as it failed to run on Windows again for some reason .. I used `mocha.opts` instead .